src/lib/Components/Bidding/Screens/ConfirmFirstTimeBid.tsx

```diff
@@ -96,8 +96,12 @@ export class ConfirmFirstTimeBid extends React.Component<ConfirmBidProps, Confir
     })
   }
 
-  onCreditCardAdded = async (params: PaymentCardTextFieldParams) => {
-    const token = await stripe.createTokenWithCard(params)
+  canPlaceBid() {
+    const { billingAddress, creditCardToken, conditionsOfSaleChecked } = this.state
+    return billingAddress && creditCardToken && conditionsOfSaleChecked
```

Comment:
Since the conditional below returns either a function or `null` if the button can't be pressed, it was redundant to write a test for this case. If we're unable to call `onPress()` in tests, it's because the fields aren't present. Thus the test that correctly stubs the state and then calls `onPress()` actually captures this as well.